starter-code/public/scripts/controllers/routes.js

```diff
@@ -1,6 +1,16 @@
 'use strict';
 var app = app || {};
 
-// TODO: Configure routes for this app with page.js, by registering each URL your app can handle, linked to a single controller function to handle it. Note that these routes do not need to wrapped in an IIFE.
+// DONE: Configure routes for this app with page.js, by registering each URL your app can handle, linked to a single controller function to handle it. Note that these routes do not need to wrapped in an IIFE.
 
-// TODO: What function do you call to activate page.js? Fire it off now, to execute. Note that it does not need to be attached to the 'app' object nor wrapped in an IIFE.
+// DONE: What function do you call to activate page.js? Fire it off now, to execute. Note that it does not need to be attached to the 'app' object nor wrapped in an IIFE.
+
+
+page('/');
+page('/about', aboutController.showAbout );
+page('/article', articleController.articleCreation);
```

Comment:
This should be the home route (`/`) not `/article` so that the articles appear on page load.